### PR TITLE
Seeds added through faker

### PR DESCRIPTION
**Pull request by alvarotrujillo**
Please validate the code !

### CODE REVIEW

**Comment by A-drienne** [comment]
NO >:(

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,38 @@
...
+puts 'Creating 50 bikes...'
+50.times do
+  bike = Bike.new(
+    name: Faker::Music::RockBand.name,
+    type: ['city bike', 'road bike', 'mountain bike'].sample,
+    location: Faker::Address.city,
+    description: Faker::Lorem,
+    gears: ['1', '3', '7', '18', '21', '28'].sample,
+    hand_brakes: [true, false].sample,
+    user_id: (1..50),
```
Comment (onofrecc) [inline db/seeds.rb]: maybe..

(1..50).sample

Comment (alvarotrujillo) [reply db/seeds.rb]: Good call, will add it now

**Review by onofrecc — COMMENTED** [review]
Amazing work

**Comment by A-drienne** [comment]
Nice! 👯‍♂️

**Pull request merged**